.travis.yml

```diff
@@ -102,173 +56,152 @@ jobs:
       before_script:
         - bash ./tests/travis/install-mysql-5.7.sh
     - stage: Test
-      php: 7.2
-      env: DB=mysql MYSQL_VERSION=5.7 COVERAGE=yes
+      php: 7.1
+      env: DB=mysqli MYSQL_VERSION=5.7
       sudo: required
       before_script:
         - bash ./tests/travis/install-mysql-5.7.sh
     - stage: Test
-      php: 7.3
-      env: DB=mysql MYSQL_VERSION=5.7
+      php: 7.1
+      env: DB=mariadb MARIADB_VERSION=10.3
+      addons:
+        mariadb: 10.3
+    - stage: Test
+      php: 7.1
+      env: DB=mariadb.mysqli MARIADB_VERSION=10.3
+      addons:
+        mariadb: 10.3
+    - stage: Test
+      php: 7.1
+      env: DB=pgsql POSTGRESQL_VERSION=11.0
```

Comment:
I have no idea why, hence my comment above:

> Travis still moves a few jobs around from time to time.

Anyway I've asked the question on the Travis Community site (they kindly ask to report issues there now, instead of using their GitHub issue tracker):

https://travis-ci.community/t/why-are-jobs-include-not-run-in-the-correct-order/1132

Reply:
Done!